src/Api/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -28,7 +28,7 @@ public static void AddSwagger(this IServiceCollection services, GlobalSettings g
             });
             config.SwaggerDoc("internal", new OpenApiInfo { Title = "Bitwarden Internal API", Version = "latest" });
 
-            config.AddSecurityDefinition("OAuth2 Client Credentials", new OpenApiSecurityScheme
+            config.AddSecurityDefinition("oauth2-client-credentials", new OpenApiSecurityScheme
```

Comment:
I assume this is openapigen not liking spaces?

Reply:
It's actually invalid according to the swagger spec.